addons/account/static/src/xml/account_payment.xml

```diff
@@ -11,25 +11,24 @@
             </t>
             <table style="width:100%;">
                 <t t-foreach="lines" t-as="line">
-                    <tr>
+                    <tr class="oe_outstanding_credit_line">
```

Comment:
No need to add this class. This bug occurs because of the clearfix on o_form_sheet.

Reply:
We have to check if we can safely removing it...

Reply:
https://github.com/odoo/odoo/pull/33863/commits/9a482fce184eddd25b63a1b3cbc0214fcc7b90fd#diff-bbeb29ba8ecd1581d7cd472402a6cc2fR159